<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidgetFrequency</class>
 <widget class="QWidget" name="SettingsWidgetFrequency">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsWidgetFrequency</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Period (hh:mm:ss)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="time_edit">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="date">
      <date>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="minimumTime">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>1</second>
      </time>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>hh:mm:ss</string>
     </property>
     <property name="currentSectionIndex">
      <number>0</number>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>1</second>
      </time>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="single_shot_check">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Single Shot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
